README.md

```diff
@@ -1,8 +1,8 @@
 # Ansible Tower/AWX Operator
```

Comment:
Maybe change title to 'Ansible AWX Operator' too to be clear? Or at least change order to "Ansible AWX/Tower Operator"